tests/pages/firefox/whatsnew/whatsnew_67_0_5.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxWhatsNew6705Page(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/67.0.5/whatsnew/all/{params}'
+
+    _firefox_accounts_form_locator = (By.ID, 'fxa-email-form')
+    _monitor_button_locator = (By.CLASS_NAME, 'js-monitor-button')
```

Comment:
This should change to `js-fxa-product-button` (that happened after this PR was made).